packages/graphback-keycloak-authz/src/KeycloakCrudService.ts

```diff
@@ -32,11 +27,11 @@ export type KeycloakCrudServiceOptions = {
 export class KeycloakCrudService<T = any> implements GraphbackCRUDService<T> {
 
   private authConfig: CrudServiceAuthConfig;
-  private wrappingService: GraphbackCRUDService;
+  private protectedService: GraphbackCRUDService;
```

Comment:
`securedService` maybe, so that to not confuse it with `protected` access level.